### DISCUSSION TITLE
How to setup Azure OpenAI models

### DISCUSSION THREAD

**Discussion by sumant-course5i** [cline/cline#648] [Feature Requests]
I've set up Cline settings this way but this throws a 404 error. These credentials work fine when I execute via script.

![Screenshot 2024-10-30 131857](https://github.com/user-attachments/assets/c6df3abc-0869-470e-91b2-c873df2d4112)

**Comment by Jasonzh8** [comment]
try this one: https://foo-bar.openai.azure.com/openai

**Reply by trstroem** [reply]
I'd go with https://foo-bar.openai.azure.com/openai/deployments/foo-gpt-4. Works for me.
